package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint ./src",
     "sync-assets": "copy-aragon-ui-assets ./public",
     "bundlewatch": "bundlewatch",
-    "prettier-ai": "./node_modules/.bin/prettier --write 'src/**/*.{ts,tsx,js}' --loglevel=error --ignore-path .prettier-imports-ignore"
```

Comment:
we don't need to point to `node_modules` here separately because by default any of the scripts will always point to node_modules itself.

Reply:
In my env it was getting the global one, idk why, but i'll remove it globally and should solve :D